phys2bids/phys2bids.py

```diff
@@ -436,6 +437,8 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                                        os.path.splitext(os.path.basename(phys_out[key].filename)
                                                         )[0]))
 
+        generate_report(outdir, logname, phys_out[key])
```

Comment:
As I wrote elsewhere in this review, I wouldn't make this mandatory, but rather optional AND suggested. That's not because I don't like the reports (I LOVE them), but because there might be occasions where you don't want them.
In that case, we need to add a parser flag, and/or make this line run ONLY if bokeh is installed.